lib/static_sync/processor.rb

```diff
@@ -46,6 +46,8 @@ def handle_conflict(file)
         if file.cached?
           raise ConflictError, "modifications to existing cached files are not allowed."
```

Comment:
But we are raising the exception here:

```
StaticSync::Processor::ConflictError: modifications to existing cached files are not allowed.
...bundler/gems/static_sync-f982792bb3f5/lib/static_sync/processor.rb:47:in `handle_conflict'
```

So somehow https://github.com/envato/static_sync/pull/1/files#diff-1b2da3445b595915666a129a9f8d5c6dL45 is matching.